public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -587,6 +590,73 @@ export const panelToRuleFormValues = async (
   return formValues;
 };
 
+export const scenesPanelToRuleFormValues = async (
+  vizPanel: VizPanel,
+  variables: SceneVariable[],
+  scenesQueries: SceneQueryRunner,
+  scenesDashboard: DashboardScene
+): Promise<Partial<RuleFormValues> | undefined> => {
+  if (!vizPanel.state.key || !scenesDashboard.state.uid) {
+    return undefined;
+  }
+
+  const relativeTimeRange = rangeUtil.timeRangeToRelative(scenesDashboard.state.$timeRange?.state.value!); //TODO: how does $timeRange work, when is $timerange undefined? when is $timeRange.state.value undefined?
+
+  const sceneq = scenesQueries.state.queries;
+
+  const queries = await dataQueriesToGrafanaQueries(
+    sceneq,
+    relativeTimeRange,
+    variables || {},
```

Comment:
This is strange, how is there used?

Reply:
ah, this wants scopedVars, which we do not need to use in scenes (as scenes have a native hierarchical variable system). scopedVars was a hacked on way to supported "local" variable overrides in the old variable system. 

But we do need to pass the scene context via scopedVars (the key way the scenes variable system is backwards compatible with the old templateSrv.replace method)